pyk/src/pyk/__main__.py

```diff
@@ -85,39 +89,44 @@ def main():
             args['output'].write('Rule: ' + rid.strip())
             args['output'].write('\nUnparsed:\n')
             args['output'].write(prettyPrintKast(rule, symbol_table))
+        _LOGGER.info(f'Wrote file: {args["output"].name}')
 
     else:
         raise ValueError(f'Unknown command: {args["command"]}')
 
 
 def create_argument_parser():
-    pykArgs = argparse.ArgumentParser()
-    pykArgs.add_argument('kompiled-dir', type=str, help='Kompiled directory for definition.')
-    pykArgs.add_argument('--verbose', '-v', default=False, action='store_true', help='Set log level to INFO.')
 
-    pykCommandParsers = pykArgs.add_subparsers(dest='command')
+    shared_options = argparse.ArgumentParser(add_help=False)
+    shared_options.add_argument('-v', '--verbose', action='count', help='Verbosity level, repeat for more verbosity.')
+
+    kompiled_definition_options = argparse.ArgumentParser(add_help=False)
+    kompiled_definition_options.add_argument('kompiled-dir', type=str, help='Kompiled directory for definition.')
```

Comment:
Lets use `--definition` or `'definition` here for consistency with `krun` and friends.

```suggestion
    kompiled_definition_args.add_argument('definition', type=str, help='Kompiled directory for definition.')
```

Reply:
Done.